bin/analyzer/dashboard.html

```diff
@@ -0,0 +1,96 @@
+
+    <!-- Bootstrap core CSS -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
+          integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+
+    <!-- Custom styles for this template -->
+    <link href="dashboard.css" rel="stylesheet">
+</head>
+
+<body>
+<nav class="navbar navbar-dark sticky-top bg-dark flex-md-nowrap p-0">
+    <a class="navbar-brand col-sm-3 col-md-2 mr-0" href="#">OPTIBENCH</a>
+    <ul class="navbar-nav px-3">
+
+    </ul>
+</nav>
+
+<div class="container-fluid">
+    <div class="row">
+        <nav class="col-md-2 d-none d-md-block bg-light sidebar">
+            <div class="sidebar-sticky">
+                <ul class="nav flex-column">
+                    <li class="nav-item">
+                        <a class="nav-link active" href="#">
+                            <span data-feather="home"></span>
+                            Dashboard <span class="sr-only">(current)</span>
+                        </a>
+
+                </ul>
+
+
+
+            </div>
+        </nav>
+
+        <main role="main" class="col-md-9 ml-sm-auto col-lg-10 pt-3 px-4">
+            <div class="d-flex justify-content-between flex-wrap flex-md-nowrap align-items-center pb-2 mb-3 border-bottom">
+                <h1 class="h2">Dashboard</h1>
+                <div class="btn-toolbar mb-2 mb-md-0">
+                </div>
+            </div>
+            <div class="alert alert-dark" role="alert">
+                Maximum throughput of <strong> {throughput} TPS </strong> has been found with a block interval of
+                <strong>{interval} seconds</strong> and <strong>{gaslimit}</strong> block gas limit.<br></br>
+
+                Total execution time for benchmarking is <strong>{executiontime}</strong> seconds
+            </div>
+            <iframe align="middle" width="600" height="600" src="bubbleplot.html"> </iframe>
+            <br></br>Above illustrated graph displays throughput for different values of gaslimit and block interval. This bubble chart with gas limit on X-axis, block interval on Y-axis represents corresponding values of throughput through size of the bubble . To know exact value of throughput at a point, hover over the bubble.
+            <br></br>
+            <iframe align="middle" width="600" height="600" src="linegraph.html"> </iframe>
+            <br></br>
+            The above graph is an interactive line plot. The drop down menu can be used to display the line plot for a particular block interval.
+            <br></br>
+            <h2>Benchmark Results</h2> 
+            Following table describes the observation from the benchmarking experiment
+            <br></br>
+            <div class="table-responsive">
+                <table class="table table-striped table-sm">
+                    <thead>
+                    {table}
```

Comment:
Can we make an interactive table somehow? I think that it would be useful to filter every column